source/parser.hera

```diff
@@ -1794,14 +1795,42 @@ FunctionExpression
       signature: {
         modifier: {},
       },
-      children: [ref, " => ", body],
+      children: [parameters, " => ", body],
       ref,
       body,
       ampersandBlock: true,
       block,
       parameters,
     }
 
+  # BinaryOp function shorthand
+  OpenParen:open !Identifier !Not BinaryOp:op CloseParen:close ->
```

Comment:
This seems like a good start. But I think we'd also like to include the `Identifier` and `Not Identifier` cases of `BinaryOp`. We'd just need to call `processBinaryOpExpression` in the body. I can work on this in a future PR though.